src/client/Microsoft.Identity.Client/Internal/Broker/BrokerSilentRequest.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.ApiConfig.Parameters;
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Internal.Requests;
+using Microsoft.Identity.Client.OAuth2;
+using Microsoft.Identity.Client.UI;
+using Microsoft.Identity.Client.Utils;
+
+namespace Microsoft.Identity.Client.Internal.Broker
+{
+    internal class BrokerSilentRequest
```

Comment:
Can you add a comment on this class stating explaining how the Android and IOS brokers use interactive / silent requests? Would also be good to add a brokers.md file in this directory with more details like: 

- how does Android broker work (from MSAL perspective) 
- how does iOS broker work from msal perspective
- account management

Reply:
Ill create a second PR when I finish writing up the Wiki page and ill add more info here and a link

Reply:
Please add comments in the code explaining the above. I don't think these details need to be in the wiki, they are too technical, but should definitely be in the code.